pilz_trajectory_generation/test/unittest_trajectory_generator_ptp.cpp

```diff
@@ -262,7 +266,18 @@ TEST_P(TrajectoryGeneratorPTPTest, testInsufficientLimit)
   sufficient_limit.has_deceleration_limits = true;
   sufficient_limit.max_deceleration = -1;
   JointLimitsContainer sufficient_joint_limits;
-  sufficient_joint_limits.addLimit(joint_name, sufficient_limit);
+  for(auto jmg : robot_model_->getJointModelGroups())
```

Comment:
I would suggest not to split it in order to avoid code duplication. I will restructure it and add comments in order to make the steps more clear. Then you can decide if you like it.